src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImportAliasCommand.cs

```diff
@@ -99,6 +100,93 @@ public SwitchParameter Force
         #endregion Parameters
 
         #region Command code
+        
+        private static bool OnlyContainsWhitespace(string line)
+        {
+            bool result = true;
+
+            foreach (char c in line)
+            {
+                if (char.IsWhiteSpace(c) && c != '\n' && c != '\r')
+                {
+                    continue;
+                }
+
+                result = false;
+                break;
+            }
+
+            return result;
+        }
+      
+        private static Collection<string> ParseCsvLine(string csv)
+        {        
+            string csvTrimmed = csv.Trim();
```

Comment:
We could use ReadOnlySpan<char>.Trim()
https://source.dot.net/#System.Private.CoreLib/shared/System/MemoryExtensions.Trim.cs,730

Reply:
I can't find a method for `ReadOnlySpan<string>.Trim()`, there seems only to be one for `char`. 
[This documentation](https://docs.microsoft.com/en-us/dotnet/api/system.memoryextensions.trim?view=netstandard-2.1) also only shows a method for `char`.

Shall I leave it like this?

Reply:
ReadOnlySpan<char> is a fairly analogous standin for a straight `string`. A `Span` (ReadOnly or not) is a sequence of data. A sequence of characters... bit like a string ;)

I haven't looked at all the surrounding code, but conceivably you could have most / all of these `string` variables changed to `ReadOnlySpan<char>` with only minor changes, not 100% sure exactly to what extent Ilya is recommending the change here.

Reply:
```powershell
        private static Collection<string> ParseCsvLine(ReadOnlySpan<char> csv) 
        {         
             ReadOnlySpan<char> csvTrimmed = csv.Trim();
```

After we move to .Net Core 3.0 we could use StreamReader.Readline() overload returned `ReadOnlySpan<char>`.

Reply:
Now we moved to .Net Core and can use StreamReader.Readline() overload returned ReadOnlySpan<char>.